pkgs/by-name/wa/wahay/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+  gnumake,
+  pkg-config,
+  gtk3,
+  git,
+  rubyPackages,
+}:
+
+buildGoModule rec {
+  pname = "wahay";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "digitalautonomy";
+    repo = "wahay";
+    rev = "v${version}";
+    hash = "sha256-59gVGDElIs+OQ/ELwOFjz6YqG18iHqoa2yYF2Ddi0/o=";
+  };
+
+  vendorHash = "sha256-w8lUnvy5dPMHWbzyyTq9Q/kE/4vSuOHffaY9CeasvQ0=";
+
+  buildInputs = [
+    gtk3
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+    gnumake
+    git
+    rubyPackages.sass
+  ];
+
+  #
+  buildPhase = ''
+    runHook preBuild
+
+    make BUILD_DIR=$out/bin TAG_VERSION=v${version} build
+
+    runHook postBuild
+  '';
+
+  checkFlags = [
+    "-skip=ConfigSuite.Test_CreateTempDir_createsTempFileInWahayDirectory"
+  ];
+
+  doCheck = false;
```

Comment:
I ran the tests myself (`make test`) and I can confirm that they do not work because they expect network access. I would however recommend to leave an explanatory comment.

Reply:
I think I properly skipped at least one of the offending tests with my latest change, but now the checkPhase is failing with `getGoDirs: command not found`, which is perplexing